src/structures/Guild.js

```diff
@@ -106,7 +106,7 @@ class Guild {
      * Whether the guild is "large" (has more than 250 members)
      * @type {boolean}
      */
-    this.large = data.large || this.large;
+    this.large = !!data.large;
```

Comment:
Can you change this to `this.large = (large in data ? !!data.large : this.large)`